frb_codegen/src/lib.rs

```diff
@@ -111,11 +111,20 @@ pub fn frb_codegen(config: &config::Opts, all_symbols: &[String]) -> anyhow::Res
         EXTRA_EXTERN_FUNC_NAMES.to_vec(),
     ]
     .concat();
+
     let c_dummy_code = generator::c::generate_dummy(&effective_func_names);
     for output in &config.c_output_path {
         fs::create_dir_all(Path::new(output).parent().unwrap())?;
+        let output_new: String = if config.count > 1 {
+            output
+                .clone()
+                .replace(".h", format!("_{}.h", config.index).as_str())
+        } else {
+            output.clone()
+        };
+
         fs::write(
-            output,
+            output_new,
             fs::read_to_string(&temp_bindgen_c_output_file)? + "\n" + &c_dummy_code,
```

Comment:
I know what the code intends to do. But there is a question here. Because `config.c_output_path` is actually  a list of ALL paths, then if there are 2 rust API blocks, with flags  ` --c-output "API_1.h" "API_2.h"` for command `flutter_rust_bridge_codegen`, then  the output file would be "API_1_0.h" "API_2_1.h", which is not expected.

More importantly, **the generated code for the 1st rust block would be overwritten** when dealing with the 2ed config ( in other word, when `frb_codegen(config, &all_symbols)` is called 2ed time in `main.rs`)

Anyway, I think the whole way to fix is not good, due to an old design issue, which I discussed [here](https://github.com/fzyzcjy/flutter_rust_bridge/pull/862#issuecomment-1362495197) in detail.